src/components/popovers/toolTip.js

```diff
@@ -281,7 +281,8 @@ class DanteTooltip extends React.Component {
             className="dante-menu-input"
             ref="dante_menu_input"
             placeholder={this.props.widget_options.placeholder}
-            onKeyPress={ this.handleInputEnter }
+            onKeyPress={this.handleInputEnter}
```

Comment:
yes, remove this and check if it's still working, it don't makes sanse to keep both